src/main/java/fitr/Fitr.java

```diff
@@ -16,18 +18,22 @@
     private ExerciseList exerciseList;
     private User user;
 
-    public Fitr(String filePathOfUserConfig, String filePathOfFoodList, String filePathOfExerciseList) {
+    public Fitr(String filePathOfUserConfig, String filePathOfFoodList, String filePathOfExerciseList,
```

Comment:
See the comments in the `Storage` class - I don't think its necessary to include a file path of the tips file since it's going to be included inside the JAR file, so can consider removing it from the constructor here..